src/components/views/donate/SwitchToAcceptedChain.tsx

```diff
@@ -10,16 +10,45 @@ import {
 import { INetworkIdWithChain } from './common.types'; // Import the type
 import { useGeneralWallet } from '@/providers/generalWalletProvider';
 import { ChainType } from '@/types/config';
+import config from '@/configuration';
 
 const SwitchToAcceptedChain: FC<{ acceptedChains: INetworkIdWithChain[] }> = ({
```

Comment:
INetworkIdWithChain has chainType as well. So, let's rely on that for switching to Solana.